src/main/resources/default/taglib/t/fileField.html.pasta

```diff
@@ -59,6 +59,7 @@
                 }
                 selectVFSFile({
                     path: value,
+                    allowUpload: true,
```

Comment:
Was this change intended?

Reply:
yes, this was previously set inside the script itself to always true. 
i would keep this here in case we might need to move it to a tag parameter or set it to false in another tag

Reply:
Okay, it just didnt match the commit message at all which hinted at a commit gone wrong